notes-webapp/src/main/webapp/vue-app/notes/components/NoteTreeviewDrawer.vue

```diff
@@ -360,7 +360,7 @@ export default {
       this.closeAll = false;
     },
     openNote(event, note) {
-      const canOpenNote = (this.filter !== this.$t('notes.filter.label.drafts') || this.filter === this.$t('notes.filter.label.drafts') && note.draftPage) && note.noteId !== this.note.id;
+      const canOpenNote = (this.filter !== this.$t('notes.filter.label.drafts') || this.filter === this.$t('notes.filter.label.drafts') && note.draftPage);
```

Comment:
This part what added one month ago by @SaraBoutej  in this commit https://github.com/Meeds-io/notes/commit/5cbf97c90dca5fdd417bdd846fef1077649951e2

@SaraBoutej can you help us to understand why you put that ? And if it is safe to remove ?
Thanks a lot

Reply:
I added this test to avoid opening a note (from the note treeview drawer) that is already opened (it was an improvement)
**For example** :if **_note-1_** is opened and when click on **_note-1_** in the tree view drawer, nothing will happen because the **_note-1_** is already opened
cc @rdenarie @abenmbarek

Reply:
so @abenmbarek can you update your modification to have the current task scenario working AND the Sara's task working ?
Thanks

Reply:
i'll check it.
thank you